src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -470,6 +470,20 @@ antlrcpp::Any CypherMainVisitor::visitIsolationLevelQuery(MemgraphCypher::Isolat
   return isolation_level_query;
 }
 
+antlrcpp::Any CypherMainVisitor::visitStorageModeQuery(MemgraphCypher::StorageModeQueryContext *ctx) {
+  auto *storage_mode_query = storage_->Create<StorageModeQuery>();
+
+  storage_mode_query->storage_mode_ = [mode = ctx->storageMode()]() {
+    if (mode->IN_MEMORY_ANALYTICAL()) {
+      return StorageModeQuery::StorageMode::IN_MEMORY_ANALYTICAL;
+    }
+    return StorageModeQuery::StorageMode::IN_MEMORY_TRANSACTIONAL;
+  }();
```

Comment:
```suggestion
  storage_mode_query->storage_mode_ = std::invoke([mode = ctx->storageMode()]() {
    if (mode->IN_MEMORY_ANALYTICAL()) {
      return StorageModeQuery::StorageMode::IN_MEMORY_ANALYTICAL;
    }
    return StorageModeQuery::StorageMode::IN_MEMORY_TRANSACTIONAL;
  });
```

Just to increase readability.